src/main/java/pl/edu/mimuw/BigInt.java

```diff
@@ -26,17 +37,90 @@ public BigInt neg() {
     return new BigInt(digits, !this.isPositive);
   }
 
+  private int readNth(int i){
+    if(i<=this.digits.length){
+      return this.digits[this.digits.length-i];
+    }
+    return 0;
+  }
+
+  private boolean biggerThan(BigInt that){
+    if (this.digits.length > that.digits.length){return true;}
```

Comment:
sorki ale te nawiasy tu nic nie wnosza (zaden formater tak nie formatuje - spokojnie mozesz je uzuwac jak to jest jedna linia, tylk orob to jednolicie)